package.json

```diff
@@ -4,6 +4,8 @@
   "description": "A2 assignment for Backend at CMDA",
   "main": "app.js",
   "scripts": {
+    "start": "node app.js",
+    "dev": "nodemon ./server.js localhost 3000",
```

Comment:
Looks good! I would suggest to put this information in the readme aswell.

Reply:
Thanks. We will add readme changes later 👍